src/components/favorite-list/favorite-list.vue

```diff
@@ -0,0 +1,53 @@
+<!--<template>
+  <List :title="'Favorite Podcasts'"  :list="podcasts" />
+</template>
+
+<script src="./script.js"></script>-->
+
+<template>
```

Comment:
Is this a copy of **list-podcast** component? why not use that component?

Reply:
My favorite-list component will call the podcast-list component, and that code is commented on the top. When @josepenagosGL integrates the favorite-button on his component, I'll uncomment that code and delete all the code remaining.